src/spectrecoff/Spectre/Layout.fs

```diff
@@ -50,7 +50,7 @@ let show (layout: Layout) =
 let hideChild identifier (layout: Layout) =
     layout[identifier].Invisible()
 
-let hide layout =
+let hide layout: Layout =
```

Comment:
If you need to specify the type, then please in brackets: `(layout: Layout)`

Reply:
Added the parentheses. 
By the way: I specified the type because during documenting I saw that the signature was `'a -> 'a`, with a being `IShowAble` (or some such).
Since this function is about Layouts though, I thought it might be better to be specific here.

Reply:
Ok, that makes sense.